### PR TITLE
Lineage pull request for: skeleton

### PR DESCRIPTION
**Pull request by cisagovbot**
# Lineage Pull Request #

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-generic.git`](https://github.com/cisagov/skeleton-generic.git)
Remote branch: `HEAD`

Check the changes in this pull request to ensure they won't cause issues with
your project.

------------

**Note:** *You are seeing this because one of this repository's maintainers has
configured [Lineage] to open pull requests.*

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

**Review by jsf9k — APPROVED** [review]
:zap:

File: README.md [hunk 1]
```diff
@@ -6,9 +6,16 @@ Status](https://github.com/cisagov/scan-target-data/workflows/build/badge.svg)](
 This is a repository for storing data used to help identify targets
 for scanning.
 
+## New Repositories from a Skeleton ##
+
+Please see our [Project Setup guide](https://github.com/cisagov/development-guide/tree/develop/project_setup)
+for step-by-step instructions on how to start a new repository from
+a skeleton. This will save you time and effort when configuring a
+new repository!
+
```
Comment (mcdonnnj) [inline README.md]: This needs to be removed from downstream projects.

**Review by mcdonnnj — CHANGES_REQUESTED** [review]
I'll fix the change I requested, just wanted to stop a merge.

**Review by jsf9k — APPROVED** [review]
Thanks @mcdonnnj.  I was wondering if the skeleton blurb was out of place in downstream projects.

**Review by mcdonnnj — APPROVED** [review]
I support these changes.

**Pull request merged**